project/Dependencies.scala

```diff
@@ -112,12 +110,10 @@ object Dependencies {
       val mockito = "org.mockito" % "mockito-core" % "3.2.4" % "test" // MIT
       // changing the scalatest dependency must be reflected in akka-docs/rst/dev/multi-jvm-testing.rst
       val scalatest = Def.setting { "org.scalatest" %% "scalatest" % scalaTestVersion.value % "test" } // ApacheV2
-      val scalatestJUnit = Def.setting { "org.scalatestplus" %% "junit-4-12" % scalaTestPlusVersion.value % "test" } // ApacheV2
-      val scalatestTestNG = Def.setting { "org.scalatestplus" %% "testng-6-7" % scalaTestPlusVersion.value % "test" } // ApacheV2
-      val scalatestScalaCheck = Def.setting {
-        "org.scalatestplus" %% "scalacheck-1-14" % scalaTestPlusVersion.value % "test"
-      } // ApacheV2
-      val scalatestMockito = Def.setting { "org.scalatestplus" %% "mockito-1-10" % scalaTestPlusVersion.value % "test" } // ApacheV2
+      val scalatestJUnit = "org.scalatestplus" %% "junit-4-12" % "3.1.0.0" % "test" // ApacheV2
+      val scalatestTestNG = "org.scalatestplus" %% "testng-6-7" % "3.1.0.0" % "test" // ApacheV2
+      val scalatestScalaCheck = "org.scalatestplus" %% "scalacheck-1-14" % "3.1.0.1" % "test" // ApacheV2
+      val scalatestMockito = "org.scalatestplus" %% "mockito-1-10" % "3.1.0.0" % "test" // ApacheV2
```

Comment:
@raboof, am I understand it right? You are adding it explicitly because in reality they are all managed independently and scala-steward can be misguided to update then all?

Reply:
Yes, AFAICT 'scalatestplus' projects are versioned with x.y.z.p where x.y.z is the scalatest version and p is the 'plus' version that's updated independently for each project. I didn't go so far to make things `scalaTestVersion + ".0"` because that'd break when we'd want to try out a scalaTest snapshot.